assignments/arrays.js

```diff
@@ -63,34 +75,96 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+function findObj(arr, id) {
+  for (const x of arr) {
```

Comment:
great job on making this function dynamic so that you not only can find the answer for car 33 but for any car a boss could for info about.